plugins/ScheduledReports/lang/en.json

```diff
@@ -10,10 +10,9 @@
         "CreateAndScheduleReport": "Create and Schedule a report",
         "CreateReport": "Create Report",
         "CustomVisitorSegment": "Custom Visitor Segment:",
-        "DescriptionOnFirstPage": "The report description will be displayed in the email message, as well as on the first page of the report.",
+        "DescriptionOnFirstPageScheduleReport": "The report description will be displayed in the schedule report message, as well as on the first page of the report.",
```

Comment:
shouldn't it be `scheduled report` in this context as well?

Reply:
Updated to `scheduled report`